### PR TITLE
[CMSP-480] mVDP Readme Update

### PR DESCRIPTION
**Pull request by jazzsequence**
Updates the README.md and readme.txt with Patchstack mVDP info.

### CODE REVIEW

**Comment by jazzsequence** [comment]
Note: behat tests are failing due to drush.in outage incident.

File: README.md [hunk 1]
```diff
@@ -276,6 +280,7 @@ Minimum supported PHP version is 7.3.
 ### Latest ###
 * Updates CONTRIBUTING.md [[#342](https://github.com/pantheon-systems/wp-saml-auth/pull/342)].
 * Bump dependencies [[#343](https://github.com/pantheon-systems/wp-saml-auth/pull/343)]
+* Updates security policy [[#349](https://github.com/pantheon-systems/wp-saml-auth/pull/349)]
```
Comment (pwtyler) [inline README.md:283]:
```suggestion
```

File: readme.txt [hunk 2]
```diff
@@ -276,6 +280,7 @@ Minimum supported PHP version is 7.3.
 = Latest =
 * Updates CONTRIBUTING.md [[#342](https://github.com/pantheon-systems/wp-saml-auth/pull/342)].
 * Bump dependencies [[#343](https://github.com/pantheon-systems/wp-saml-auth/pull/343)]
+* Updates security policy [[#349](https://github.com/pantheon-systems/wp-saml-auth/pull/349)]
```
Comment (pwtyler) [inline readme.txt:283]:
```suggestion
```

**Review by pwtyler — DISMISSED** [review]
Per recent conversation, avoid adding asset-only or process-type changes to the README changelogs.

**Comment by jazzsequence** [comment]
Yolo'ing this.

**Pull request merged**